shared/crossCompilation/_default.md

```diff
@@ -0,0 +1,3 @@
+#### Setting up {{ $language.name }} cross-compilation
```

Comment:
If we deploy this now the C++ example says that cross-compilation is not necessary, so I think we need to deploy the changes for that together with this one.